CRM/Contribute/Form/Task/PDF.php

```diff
@@ -187,6 +195,14 @@ public function postProcess() {
       $objects['contribution']->receive_date = CRM_Utils_Date::isoToMysql($objects['contribution']->receive_date);
 
       $values = array();
+      if (isset($params['fromEmailAddress']) && !$elements['createPdf']) {
+        $fromEmail = $params['fromEmailAddress'];
+        $from = CRM_Utils_Array::value($fromEmail, $this->_emails);
+        $fromDetails = explode(' <', $from);
```

Comment:
No, I meant, as we are already converting and storing `$fromDetails` at line 157-159. Do we require to write these lines again? As I find this to be working if I remove the above three lines.